lib/rules/time-min-milliseconds/index.js

```diff
@@ -0,0 +1,80 @@
+"use strict"
+
+const _ = require("lodash")
+const declarationValueIndex = require("../../utils/declarationValueIndex")
+const report = require("../../utils/report")
+const ruleMessages = require("../../utils/ruleMessages")
+const validateOptions = require("../../utils/validateOptions")
+const keywordSets = require("../../reference/keywordSets")
+const postcss = require("postcss")
+const valueParser = require("postcss-value-parser")
+
+const ruleName = "time-min-milliseconds"
+
+const messages = ruleMessages(ruleName, {
+  expected: time => `Expected no less than ${time} milliseconds`,
```

Comment:
Let's say `Expected of a minimum of ${time} milliseconds` to avoid quarrels about "less" vs "fewer".